mobile/evidence/ios/Features/Sources/Login/LoginError.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  File.swift
+//  
+//
+//  Created by Cris Messias on 16/05/24.
+//
+
+import Foundation
+
+enum LoginError: Error {
+    case emailOrPasswordNotProvide
+    case emailNotProvide
+    case invalidEmailOrPassword
+    case invalidEmail
+    case emailAlreadyInUse
+    case weakPassword
+    case userNotFound
+    case networkError
+    case internalError
+    case tooManyRequests
+    case userDisabled
+    case userAuthenticationFailed
+    case unknown
+
+    var errorDescription: String? {
+        switch self {
+        case .emailOrPasswordNotProvide: 
+            "Email or password not provided."
+        case.emailNotProvide:
+            "Email not provided."
+        case.invalidEmailOrPassword:
+            "Email or password not valid."
+        case.invalidEmail:
+            "The email address is badly formatted."
+        case.emailAlreadyInUse:
+            "The email address is already in use by another account."
+        case .userNotFound:
+            "There is no user record corresponding to this identifier. The user may have been deleted."
+        case .weakPassword:
+            "The password must be 6 characters long or more."
```

Comment:
Have you found documentation about this error? Is the length the only cause?

Reply:
That's what I found in their documentation:
No, if also has characters or numbers and letters.


Indica uma tentativa de definição de uma senha considerada muito fraca. O campo NSLocalizedFailureReasonErrorKey no objeto de dicionário NSError.userInfo conterá uma explicação mais detalhada que pode ser exibida ao usuário.

https://firebase.google.com/docs/auth/ios/errors?hl=pt-br

Reply:
But, I don't think this error is showing on the screen. 

I have this function that to use the `inputValidator`, I think this firebase is not being called.

`func errorMessage() -> String? {
        if isCreateAccountButtonPressed && (emailInput.isEmpty){
            return LoginError.emailNotProvide.errorDescription ?? nil
        } else if isCreateAccountButtonPressed == true && (isValidEmail == false) {
            return LoginError.invalidEmail.errorDescription ?? nil
        } else 
        if let message = createMessageError, isCreateAccountButtonPressed {
            return message
        }
        return nil
    }`

Reply:
In the end, I am not using this case.